mesonbuild/compilers/rust.py

```diff
@@ -121,6 +121,11 @@ def get_sysroot(self) -> str:
         p, stdo, stde = Popen_safe_logged(cmd)
         return stdo.split('\n', maxsplit=1)[0]
 
+    def get_crt_static(self) -> bool:
```

Comment:
This looks worthwhile to cache, adding an `@functools.lru_cache(maxsize=None)` seems worthwhile to avoid calling a sub-process again.